### DISCUSSION TITLE
API route returns favicon.ico

### DISCUSSION THREAD

**Discussion by Jahrhause** [vercel/next.js#43361] [Help] [answered]
Hi

Im trying to to use api routes to return an Image from our own image CDN. and afterwards use it with Next image.
The response returns the image but also a favicon.ico, which I think automatically comes from Next and this causes errors.
 
Does anybody know how to not make it return the favicon?
Heres my code

```
export default async function handler(req: NextApiRequest, res: NextApiResponse) {
  try {
    res.setHeader(
      'Cache-Control',
      `public, immutable, no-transform, s-maxage=31536000, max-age=31536000`
    );
    res.setHeader('content-type', 'image/jpeg');

    const image = await fetch('URL to CDN');

    const contentType = image.headers.get('content-type') || '';
    const body = image.body;

    res.setHeader('content-type', contentType);

    return res.status(200).send(body);
  } catch (error) {
    console.error('Error when retrieving image from API');
    return res.status(404).end();
  }
}
```

**Comment by icyJoseph** [comment]
How do your routes look like? can you do a `tree src/pages` or (pages)

**Reply by Jahrhause** [reply]
Hi @icyJoseph 

Found the error in my image component where i had misspelled the API path :(

**Comment by Jahrhause** [comment, marked as answer]
Misspelling caused the error
